docs/examples/container-config.json

```diff
@@ -3,7 +3,7 @@
     "name": "busybox"
   },
   "image":{
-    "image": "busybox"
+    "image": "index.docker.io/busybox"
```

Comment:
index is added back again?

Reply:
@feiskyer fixed (again); sorry - this is embarrassing - that's what you get when accepting Suggested Changes on GitHub's UI and forget to pull that locally - making further changes causes you to loose those fixes again. 😈 Merge now?